src/Files.App.Controls/Files.App.Controls.csproj

```diff
@@ -15,6 +15,33 @@
         <Optimize Condition="'$(Configuration)' != 'Debug'">true</Optimize>
     </PropertyGroup>
 
+	<PropertyGroup>
+		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+		<Title>Files UI Controls</Title>
+		<Authors>yaira, files-community, mdta</Authors>
+		<Company>Files Community</Company>
+		<Copyright>Copyright (c) 2024 Files Community</Copyright>
+		<PackageReadmeFile>README.md</PackageReadmeFile>
+		<RepositoryUrl>https://github.com/files-community/Files</RepositoryUrl>
+		<RepositoryType>git</RepositoryType>
+		<PackageTags>files, files-community</PackageTags>
+		<Description>Stunningly designed UI controls to enhance the experience of organizing files and folders.</Description>
```

Comment:
Elevate your WinUI applications with our collection of custom-built controls, crafted specifically to address our needs in Files. These controls are provided "as is," with no guaranteed support, but we hope they prove useful to the developer community.